data/aecore/.ceres/hc_smart_contract_accounts.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Should this be identical to `hc_smart_contrac_accounts_test.json`?

Reply:
Yes. They should be generated according the network ID. This is already done in a different branch. Anyway, I tracked it in an issue #3974